lib/utils.js

```diff
@@ -22,6 +22,37 @@ export function calculateSlippageAmount(value) {
   return value.mul(SLIPPAGE_PERCENTAGE).div(100)
 }
 
+export function useCheckEmailForAddress() {
+  const [userExists, setUserExists] = useState(false)
+  const { account: address } = useWeb3Connect() || ''
+  return useCallback(async () => {
+    try {
+      const response = await fetch(`${env('BACKEND_URL')}/user/${address}`)
+      if (!response.ok) {
+        throw new Error(
+          `Received a wrong status when trying to fetch if user exists: ${response.status}.`
+        )
+      }
+      console.log(response)
+    } catch (err) {
+      console.error(
+        `Error while trying to fetch if user exists ${address}`,
+        err
+      )
```

Comment:
Should we move this after the environment check?

Reply:
I actually think it makes more sense to do it _before_, so that when the "ignoring on development" call is seen, we can just dismiss this.